ui/apps/platform/cypress/integration/collections/collectionsCrudWorkflow.test.js

```diff
@@ -6,7 +6,7 @@ const baseUrl = '/v1/collections';
 const autocompleteUrl = `${baseUrl}/autocomplete`;
 
 // Cleanup an existing collection via API call
-function tryDeleteCollection(collectionName) {
```

Comment:
Temporary, pending rebase, or outdated change?

The function moves to helpers after you merge #4251

Reply:
Temporary + pending rebase after merging 4251 :)